extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -2621,11 +2556,6 @@ public String getCurrentZoneName() {
         return stateMap.get("CurrentZoneName");
     }
 
-    public String getCurrentURIFormatted() {
```

Comment:
I was just not sure if it is called in handleCommand() linked to one of the Channels of the Thing